playwright/e2e/crypto/crypto.spec.ts

```diff
@@ -453,8 +456,8 @@ test.describe("Cryptography", function () {
             // no e2e icon
             await expect(lastTileE2eIcon).not.toBeVisible();
 
-            // It can take up to 10 seconds for the key to be backed up. We don't really have much option other than
-            // to wait :/
+            // Workaround for https://github.com/element-hq/element-web/issues/27267: it can take up to 10 seconds for
```

Comment:
Nit (the `:` is merged with the url)

```suggestion
            // Workaround for https://github.com/element-hq/element-web/issues/27267 : it can take up to 10 seconds for
```

Reply:
Let's try:

```suggestion
            // Workaround for https://github.com/element-hq/element-web/issues/27267. It can take up to 10 seconds for
```